aws/scenarios/vm/os/os.go

```diff
@@ -32,8 +32,8 @@ func GetOS(env aws.Environment, osType Type) (OS, error) {
 		return newWindows(env), nil
 	case UbuntuOS:
 		return newUbuntu(env), nil
-	case MacosOS:
-		return newMacOS(env), nil
+	//case MacosOS:
+	//return newMacOS(env), nil // Not yet supported
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
What about returning a specific error instead ?
```suggestion
	case MacosOS:
	    return nil, errors.New("MacosOS is not supported yet")
```

Reply:
I want to avoid someone writing code for MacOS and realise later it is not supported.